CHANGES.md

```diff
@@ -10,6 +10,7 @@ git version
     - handle `=` syntax in compiler flags (#1409)
     - expose all destruct exceptions in the api (#1437)
     - fix superfluous break in error reporting (#1432)
+    - recognise binding operators in locate and occurrences (#1398)
```

Comment:
```suggestion
    - recognise binding operators in locate and occurrences (#1398, @mattiase)
```

Reply:
Not really fond of tooting my own horn but you make the rules; changing.

Reply:
We usually credit external contributors, but don't feel obliged to do it if you prefer to remain in the shadows :-)

Reply:
> We usually credit external contributors, but don't feel obliged to do it if you prefer to remain in the shadows :-)